.gitignore

```diff
@@ -1 +1,3 @@
-.env
+**/.env
+data
```

Comment:
To make sure all .env files in all directories are ignored in git. Committing a .env file accidentally would be bad. Also, I've been using `./data` as my test directory rather than my media directory, so I decided to put that in as well.

Reply:
Yeah my `?` was about the `data` bit that does not make much sense to me. But I don't have any strong feelings about removing it if you find it useful 👍